primitives/types/module_test.go

```diff
@@ -0,0 +1,77 @@
+package types
+
+import (
+	"strconv"
+	"testing"
+
+	sc "github.com/LimeChain/goscale"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	numModules = 4
+)
+
+var (
+	mockAuraModule      Module
+	mockSystemModule    Module
+	mockGrandpaModule   Module
+	mockTxPaymentModule Module
+	modules             []Module
+)
+
+type module struct {
+	Module
+	Index sc.U8
+}
+
+func New(index sc.U8) Module {
```

Comment:
Let's rename this to something related to modules as all other tests can see it.